### ISSUE TITLE
[Enhancement] strlen misused does not recognise > 0

### ISSUE THREAD

**Issue by andreasschroth** [kalessil/phpinspectionsea#1061]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Enhancement                          |
| Plugin         | Php Inspections (EA Extended), 3.0.6.1 |
| Language level | PHP 7.2                        |

Current behaviour:
---

Here is some example code:

```php
function checkInput(string $input): string
{
    if (strlen($input) > 0) {
        return 'Input is not empty';
    }
    return 'Input is empty';
}
```

Right now, no warning from the "strlen misused" inspection. If the `if` is changed to `if (strlen($input) !== 0) { ... }`, the inspection works.

Expected behaviour:
---

I would expect the inspection to warn about the above if statement and suggest to replace it by `if ($input !== '') { ... }`.

**Comment by kalessil** [comment]
Fixed!
